### ISSUE TITLE
New-AWSPowerShellLambdaPackage fails when output path contains a space

### ISSUE THREAD

**Issue by ndabas** [aws/aws-lambda-dotnet#533]
Like the title says, `New-AWSPowerShellLambdaPackage` fails when the output path contains a space. It accepts a script name with a space in the path though.

Actually, the output zip file does get created -- if I specify `/home/new folder/test.zip` as the output path, it gets created at `/home/new.zip`.

A full shell transcript is below. This was run in Docker with .NET Core SDK 2.1 and PowerShell 6.2.3.

```
PS /home/new folder> New-AWSPowerShellLambda -ScriptName MyFirstPSScript -Template Basic
Configuring script to use installed version 3.3.590.0 of AWSPowerShell.NetCore.
Created new AWS Lambda PowerShell script MyFirstPSScript.ps1 from template Basic at /home/new folder/MyFirstPSScript
PS /home/new folder> New-AWSPowerShellLambdaPackage -ScriptPath ./MyFirstPSScript/MyFirstPSScript.ps1 -OutputPackage ./M
yFirstPSScript/MyFirstPSScript.zip -Verbose
Staging deployment at /tmp/MyFirstPSScript
Configuring PowerShell to version 6.1.1
Generating C# project /tmp/MyFirstPSScript/MyFirstPSScript.csproj used to create Lambda function bundle.
Generating /tmp/MyFirstPSScript/Bootstrap.cs to load PowerShell script and required modules in Lambda environment.
Generating aws-lambda-tools-defaults.json config file with default values used when publishing project.
Copying PowerShell script to staging directory
VERBOSE: Populating RepositorySourceLocation property for module AWSPowerShell.NetCore.
VERBOSE: Loading module from path '/root/.local/share/powershell/Modules/AWSPowerShell.NetCore/3.3.590.0/ImportGuard.ps1'.
VERBOSE: Loading module from path '/root/.local/share/powershell/Modules/AWSPowerShell.NetCore/3.3.590.0/AWSPowerShellCompleters.psm1'.
VERBOSE: Loading module from path '/root/.local/share/powershell/Modules/AWSPowerShell.NetCore/3.3.590.0/AWSPowerShellLegacyAliases.psm1'.
VERBOSE: Loading module from path '/root/.local/share/powershell/Modules/AWSPowerShell.NetCore/3.3.590.0/AWSPowerShell.NetCore.dll'.
VERBOSE: Found installed module AWSPowerShell.NetCore (3.3.590.0) to save with package bundle.
Copying local module AWSPowerShell.NetCore(3.3.590.0) from /root/.local/share/powershell/Modules/AWSPowerShell.NetCore/3.3.590.0
Resolved full output package path as /home/new folder/./MyFirstPSScript/MyFirstPSScript.zip
Creating deployment package at /home/new folder/./MyFirstPSScript/MyFirstPSScript.zip
Restoring .NET Lambda deployment tool
VERBOSE: Installing .NET Global Tool Amazon.Lambda.Tools
VERBOSE: Error installing, attempting to update Amazon.Lambda.Tools
VERBOSE: Looking for windows excutable for dotnet-lambda.exe
VERBOSE: Did not find windows executable, assuming on non windows platform and using dotnet-lambda
Initiate packaging
VERBOSE: ~/.dotnet/tools/dotnet-lambda package  --configuration Release --framework netcoreapp2.1 --function-runtime dotnetcore2.1 --output-package /home/new folder/./MyFirstPSScript/MyFirstPSScript.zip
VERBOSE: Amazon Lambda Tools for .NET Core applications (3.3.1)
VERBOSE: Project Home: https://github.com/aws/aws-extensions-for-dotnet-cli, https://github.com/aws/aws-lambda-dotnet
VERBOSE:
VERBOSE: Executing publish command
VERBOSE: ... invoking 'dotnet publish', working folder '/tmp/MyFirstPSScript/bin/Release/netcoreapp2.1/publish'
VERBOSE: ... Disabling compilation context to reduce package size. If compilation context is needed pass in the "/p:PreserveCompilationContext=false" switch.
VERBOSE: ... publish: Microsoft (R) Build Engine version 16.2.32702+c4012a063 for .NET Core
VERBOSE: ... publish: Copyright (C) Microsoft Corporation. All rights reserved.
VERBOSE: ... publish:   Restore completed in 980.85 ms for /tmp/MyFirstPSScript/MyFirstPSScript.csproj.
VERBOSE: ... publish:   MyFirstPSScript -> /tmp/MyFirstPSScript/bin/Release/netcoreapp2.1/rhel.7.2-x64/MyFirstPSScript.dll
VERBOSE: ... publish:   MyFirstPSScript -> /tmp/MyFirstPSScript/bin/Release/netcoreapp2.1/publish/
VERBOSE: Changed permissions on published file (chmod +rx Microsoft.PowerShell.MarkdownRender.dll).
...
VERBOSE: Changed permissions on published file (chmod +rx Microsoft.Win32.SystemEvents.dll).
VERBOSE: Zipping publish folder /tmp/MyFirstPSScript/bin/Release/netcoreapp2.1/publish to /home/new
VERBOSE: ... zipping: updating: Microsoft.PowerShell.MarkdownRender.dll (deflated 57%)
...
VERBOSE: ... zipping: updating: Modules/AWSPowerShell.NetCore/3.3.590.0/AWSSDK.MediaStore.dll (deflated 69%)
VERBOSE: Created publish archive (/home/new).
VERBOSE: Lambda project successfully packaged: /home/new
When deploying this package to AWS Lambda you will need to specify the function handler. The handler for this package is
: MyFirstPSScript::MyFirstPSScript.Bootstrap::ExecuteFunction. To request Lambda to invoke a specific PowerShell functio
n in your script specify the name of the PowerShell function in the environment variable AWS_POWERSHELL_FUNCTION_HANDLER
 when publishing the package.
Resolve-Path : Cannot find path '/home/new folder/MyFirstPSScript/MyFirstPSScript.zip' because it does not exist.
At /root/.local/share/powershell/Modules/AWSLambdaPSCore/1.2.0.0/Public/New-AWSPowerShellLambdaPackage.ps1:191 char:47
+ ...   PathToPackage                   = Resolve-Path -Path $OutputPackage
+                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (/home/new folder/My\u2026MyFirstPSScript.zip:String) [Resolve-Path], ItemNotF
oundException
+ FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.ResolvePathCommand


VERBOSE: Removing staging directory /tmp/MyFirstPSScript
LambdaHandler                                               PathToPackage PowerShellFunctionHandlerEnvVar
-------------                                               ------------- -------------------------------
MyFirstPSScript::MyFirstPSScript.Bootstrap::ExecuteFunction               AWS_POWERSHELL_FUNCTION_HANDLER

PS /home/new folder> ls -l /home
total 27912
drwxr-xr-x 2 root root     4096 Oct  7 21:12 MyFirstPSScript
drwxr-xr-x 3 root root     4096 Oct  7 21:02 new folder
-rw-r--r-- 1 root root 28571865 Oct  7 21:13 new.zip

```

**Comment by klaytaybai** [comment]
Thanks for reporting this

**Comment by ndabas** [comment]
I have submitted a PR to fix this: https://github.com/aws/aws-lambda-dotnet/pull/535

**Comment by NGL321** [comment]
Closed by #535
